Package.swift

```diff
@@ -16,19 +16,21 @@ let package = Package(
     name: "SpeziOnboarding",
     defaultLocalization: "en",
     platforms: [
-        .iOS(.v17),
-        .visionOS(.v1),
-        .macOS(.v14)
+        .iOS(.v18),
+        .visionOS(.v2),
+        .macOS(.v15)
```

Comment:
@PSchmiedmayer these changes are bc parts of the new code use typed throws, which require the iOS18/equiv runtime. we could change the API to not use typed throws, in which case we should probably be able to keep the old deployment targets

Reply:
In general, can we start requiring iOS 18 for some of our packages? E.g. `Synchronization` would be nice in SpeziLLM sometimes 😄